lib/samples/query_feature_table/query_feature_table_sample.dart

```diff
@@ -50,42 +55,50 @@ class _QueryFeatureTableSampleState extends State<QueryFeatureTableSample> {
   Widget build(BuildContext context) {
     return Scaffold(
       resizeToAvoidBottomInset: false,
-      body: Column(
-        children: [
-          TextField(
-            focusNode: _searchFocusNode,
-            controller: _textEditingController,
-            decoration: InputDecoration(
-              hintText: 'Enter a State...',
-              prefixIcon: const Icon(Icons.search),
-              suffixIcon: IconButton(
-                onPressed: dismissSearch,
-                icon: const Icon(Icons.clear),
+      // create a column with a text field and a map view.
+      body: SafeArea(
+        child: Column(
+          children: [
+            // create a text field for searching states.
+            TextField(
+              focusNode: _searchFocusNode,
+              controller: _textEditingController,
+              decoration: InputDecoration(
+                hintText: 'Enter a State...',
+                prefixIcon: const Icon(Icons.search),
+                suffixIcon: IconButton(
+                  onPressed: dismissSearch,
+                  icon: const Icon(Icons.clear),
+                ),
               ),
+              onSubmitted: onSearchSubmitted,
             ),
-            onSubmitted: onSearchSubmitted,
-          ),
-          Expanded(
-            child: ArcGISMapView(
-              controllerProvider: () => _mapViewController,
-              onMapViewReady: onMapViewReady,
+            // add a map view to the widget tree and set a controller.
+            Expanded(
+              child: ArcGISMapView(
+                controllerProvider: () => _mapViewController,
+                onMapViewReady: onMapViewReady,
+              ),
             ),
-          ),
-        ],
+          ],
+        ),
       ),
     );
   }
 
   void onMapViewReady() async {
+    // create a map with the topographic basemap style and set to the map view.
     final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
 
+    // create a feature table and a feature layer.
```

Comment:
I think we can define the feature table as an instance variable and I don't think we need to explicitly load it as we don't need to access any attributes before we add the feature layer to the map, and there is automatic loading done on these loadables. I can talk through this a bit more if this doesn't make sense.

So instead do:

```
// create a feature table and a feature layer.
final _featureTable = ServiceFeatureTable.withUri(Uri.parse('https://services.arcgis.com/jIL9msH9OI208GCb/arcgis/rest/services/USA_Daytime_Population_2016/FeatureServer/0'));
late FeatureLayer _featureLayer;
```